ceph-salt-formula/salt/ceph-salt/container.sls

```diff
@@ -14,6 +16,8 @@
     - mode: '0644'
     - backup: minion
     - failhard: True
+  {% endif %}
```

Comment:
```suggestion
    - defaults:
        registries: {{ registries }}
  {% endif %}
```
I got this error when testing:
```[2020-05-04 05:33:47.825560] [master.mini.com ] Finished with failures

Failure in minion: master.mini.com
__id__: /etc/containers/registries.conf
__run_num__: 6
__sls__: ceph-salt.container
changes: {}
comment: 'Unable to manage file: Jinja variable ''registries'' is undefined'
duration: 40.112
name: /etc/containers/registries.conf
result: false
start_time: '07:33:47.759825'
state: file_|-/etc/containers/registries.conf_|-/etc/containers/registries.conf_|-managed
```

The variable `registries` seems not brought into template automatically.

Reply:
Done